api/src/middlewares/authoriseRequest.ts

```diff
@@ -1,27 +1,26 @@
 import { Response, Request, NextFunction } from "express";
 import { Scope, UserAttributes } from "../utils/types/attributeTypes";
-import { getRequiredScopes } from "../utils/RBAC";
+import Logger from "../utils/Logger";
 
 export const authoriseRequest = async (
-  req: Request,
+  _: Request,
   res: Response,
   next: NextFunction
 ) => {
   // & unknown to indicate possible presence of other data. Does not affect typing, only documentative.
   const user: (UserAttributes & unknown) | undefined = res.locals.user;
   if (!user) {
     // Set within authenticateRequest middleware
-    console.log("authoriseRequest middleware called before authenticateRequest middleware");
+    Logger.error("Missing user within authoriseRequest from authenticateRequest.");
     res.status(500).end();
```

Comment:
C: Could we use the ErrorController to throw an exception then have the NextFunction Express middleware handle it?

Reply:
Potentially, though I would wonder whether calling `next` within `authoriseRequest` would pass that value into the controller, or into `errorHandler` (which I think it would) - not familiar enough with `express` to say with confidence.

Reply:
Yeah, that's an interesting question. I think if the error is throw it will only pass the Error in. What do you make of the docs for it: https://expressjs.com/en/guide/error-handling.html

Reply:
From the docs, calling `next(error)` will skip all middleware that is not setup as error handling, so this could throw an error which will only be processed by `errorHandler`.